src/helpers/library.ts

```diff
@@ -185,16 +185,12 @@ async function uninstall({
     const wineprefix = (await getGameSettings(appName, runner)).winePrefix
 
     linuxArgs = {
-      checkboxLabel: [
-        t(
-          'gamepage:box.uninstall.checkbox',
-          "Would you like to remove the prefix aswell? This can't be undone."
-        ),
-        `${t(
-          'gamepage:box.uninstall.checkbox_prefix',
-          'Prefix'
-        )}: ${wineprefix}`
-      ].join('\n'),
+      checkboxLabel: t('gamepage:box.uninstall.checkbox', {
+        defaultValue:
+          "Remove prefix: {{prefix}}{{newLine}}Note: This can't be undone and will also remove not backed up save files.",
+        prefix: wineprefix,
+        newLine: '\n'
```

Comment:
This feels a little weird, why not just put the newline into the `defaultValue` directly?

Reply:
I don't know how webplate or i18n handles this. Maybe it shows the \n in the translation and break something?
With the variable i know webplate and i18n can handle this. Didn't find a useful entry if `\n` is supported.

Reply:
*I think* `\n` is handled properly, at least some other parts of the Weblate docs mention it
But doing it like this with the variable also works of course

Reply:
I don't mind. Can add the \n. Woould be good to know how the translator prefer it.